recipes/libsystemd/all/conanfile.py

```diff
@@ -70,14 +71,19 @@ def validate(self):
             raise ConanInvalidConfiguration("Only Linux supported")
 
     def build_requirements(self):
-        self.tool_requires("meson/1.2.2")
+        self.tool_requires("meson/1.3.0")
         self.tool_requires("m4/1.4.19")
         self.tool_requires("gperf/3.1")
         if not self.conf.get("tools.gnu:pkg_config", check_type=str):
-            self.tool_requires("pkgconf/2.0.3")
+            self.tool_requires("pkgconf/2.1.0")
 
     def source(self):
-        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        # Extract using standard Python tools due to Conan's unzip() not handling backslashes in
+        # 'units/system-systemd\x2dcryptsetup.slice', etc. correctly.
+        download(self, **self.conan_data["sources"][self.version], filename="sources.tar.gz")
```

Comment:
Let's check in the client if anything can be done about fixing this behaviour :)